src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Daniel Zerbino <notifications@github.com> writes:
> In src/main/resources/avro/sequenceAnnotations.avdl:
> 
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
> 
> Good point, a transcript can be assigned multiple parent genes (e.g. fusion
> transcript, trans-spliced, etc)

This can be done with either multiple parents or with children.
I think either representation can store the data.  So please
send explanations as to why one is better.  Will add rational to
documentation and see what the masses have to say.